src/encoded/static/components/viz/QuickInfoBar.js

```diff
@@ -33,6 +33,7 @@ export default class QuickInfoBar extends React.PureComponent {
         // If have href, only show for /browse/, /search/, and / & /home
         var urlParts = url.parse(href);
         if (urlParts.hash && urlParts.hash.indexOf('!impersonate-user') > -1) return true;
+        if (urlParts.search && urlParts.search.indexOf('currentAction=impersonate-user') > -1) return true;
```

Comment:
It looks like we're keeping both methods of impersonate-user here - is that "just in case" users can impersonate in 2 different ways?

Reply:
Agreed, seems like `if (urlParts.hash && urlParts.hash.indexOf('!impersonate-user') > -1) return true;` can be removed

Reply:
Same comment as above -- I think both checks for impersonat user can be removed since that is now done from the user page

Reply:
Yep was just in case of... of I don't know what actually lol.
At the last minute changed URL path of impersonate user form from `/?currentAction=impersonate-user` to `/users/currentuseruuid/?currentAction=impersonate-user` so removed.